lookbook/docs/patterns/18-aria-live.md.erb

```diff
@@ -0,0 +1,93 @@
+## What are ARIA Live Regions?
+ARIA Live Regions are a crucial accessibility feature that allows screen readers to announce dynamic content updates without requiring user focus changes.
+
+### ARIA Live Attributes
+ARIA Live Regions rely on specific attributes to control how updates are announced:
+- `aria-live="polite"`: Announces updates when the user is idle.
+- `aria-live="assertive"`: Announces updates immediately, interrupting the user.
+- `aria-atomic="true"`: Ensures the entire content of the region is read when it changes.
+
+## Implementing ARIA Live Regions in Primer
+Primer provides a built-in component named live-region to implement ARIA Live Regions easily.
+We have decided to use a single global ARIA live region in base.html to ensure live regions are present in the DOM before updates occur. This approach is better than having multiple live regions because it ensures that announcements are delivered in a clear and consistent manner. By centralizing updates, we avoid potential conflicts between overlapping live regions and provide a more seamless experience for screen reader users.
+There is an exception for modal or dialog, especially if that dialog traps focus. When a dialog is open, screen readers often limit their reading and focus to within the dialog. If aria-live region is in base.html (outside the dialog), it becomes “invisible” to the screen reader while the modal is open — it's still updated in the DOM, but not announced until the dialog is closed, and the screen reader returns to the full DOM scope.
+In this case, we should add the live-region component to the dialog itself, like what we have implemented for primerized date picker.
```

Comment:
Maybe you should make this a separate section with its own heading (e.g. "Live regions in Dialogs") and push it a bit down, maybe even at the end. It feels very confusing and I am missing an example.